packages/component/src/BasicTranscript.js

```diff
@@ -45,28 +51,24 @@ const ROOT_CSS = css({
   }
 });
 
-function useMemoize(fn) {
-  return useMemo(() => {
-    let cache = [];
+function intersectionOf(arg0, ...args) {
+  return args.reduce(
+    (interim, arg) =>
+      interim.reduce((intersection, item) => {
+        arg.includes(item) && intersection.push(item);
 
-    return run => {
-      const nextCache = [];
-      const result = run((...args) => {
-        const { result } = [...cache, ...nextCache].find(
-          ({ args: cachedArgs }) =>
-            args.length === cachedArgs.length && args.every((arg, index) => Object.is(arg, cachedArgs[index]))
-        ) || { result: fn(...args) };
-
-        nextCache.push({ args, result });
-
-        return result;
-      });
+        return intersection;
+      }, []),
+    arg0
+  );
+}
 
-      cache = nextCache;
+function removeInline(array, ...items) {
+  items.forEach(item => {
+    const index = array.indexOf(item);
 
-      return result;
-    };
-  }, [fn]);
+    ~index && array.splice(index, 1);
+  });
```

Comment:
If the array contains multiple instances of an item, this will only remove the first. Also, `~x && y` is not very legible.

Reply:
Moved `removeInline` to a separate `removeInline.js`, added tests, and also deal with multiple occurrences.